src/validate.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable max-lines */
+function validate(targetWord, toCase) {
+  const cases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
```

Comment:
Better way use SUPPORTED_CASES ( or something like that) for constant name.
Also, move all constants (status code, error messages) to upper scope or better way - to separate file.